app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -420,8 +420,6 @@ function AcuantCapture(
     if (event.target === inputRef.current) {
       const isAcuantCaptureCapable = hasCapture && !acuantFailureCookie;
       const isEnvironmentCapture = capture !== 'user';
-      const shouldStartSelfieCapture =
-        isAcuantLoaded && capture === 'user' && !isForceUploading.current;
```

Comment:
We can probably remove the `capture` prop altogether, since it's essentially controlling the facing mode of the camera, and now we only have the one facing mode ("environment").

Reply:
Same note applies to the `capture` prop on the `FileInput` component.